src/main/java/application/rest/v1/ComponentInfoRegistry.java

```diff
@@ -284,16 +287,16 @@ private void processGroupVersion(ApiClient client, Map<String,ComponentInfo> map
                             String kind = resource.get("kind").getAsString();
                             String key = (group != null ? group : "") + "/" + version + "/" + kind;
                             String groupKindKey = (group != null ? group : "") + "/" + kind;
-                            System.out.println("processGroupVersion ComponentInfo map key: " + key);
+                            Logger.log(className, "processGroupVersion", Logger.LogType.INFO, "ComponentInfo map key: " + key);
```

Comment:
This group is INFO messages but I'm not sure why.  They look more like debug.

Reply:
I changed it to use DEBUG.